### PR TITLE
Doc update for sdc and other version bumpup

### PR DESCRIPTION
**Pull request by adarsh-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1012|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -164,8 +164,6 @@ Use the below command to replace or update the secret:
    ```bash
    cd dell-csi-helm-installer && wget -O myvalues.yaml https://github.com/dell/helm-charts/raw/csi-vxflexos-2.9.0/charts/csi-vxflexos/values.yaml
    ```  
-   >Note: To connect to a PowerFlex 4.5 array, edit the powerflexSdc parameter in your values.yaml file to use dellemc/sdc:4.5:
```
Comment (adarsh-dell) [inline content/docs/csidriver/installation/helm/powerflex.md:167]: Removing this note since, starting from CSM 1.9 and v2.9.0 of the driver, SDC will be present in the sample file by default. For CSM 1.8 and v2.8.0 of the driver, it will be located under the v2 directory.

**Review by suryagupta4 — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]
lgtm

**Review by mareksuski-dell — DISMISSED** [review]
lgtm

File: content/docs/csidriver/installation/offline/_index.md [hunk 2]
```diff
@@ -94,9 +94,9 @@ cd dell-csi-operator/scripts
    dellemc/csi-vxflexos:v2.8.0
    dellemc/csi-vxflexos:v2.9.0
    dellemc/dell-csi-operator:v1.12.0
-   dellemc/sdc:3.6
    dellemc/sdc:3.6.0.6
```
Comment (gallacher) [inline content/docs/csidriver/installation/offline/_index.md:97]: What about the removal of the other dellemc/sdc:* values?

File: content/docs/csidriver/installation/offline/_index.md [hunk 3]
```diff
@@ -225,9 +225,9 @@ Preparing a offline bundle for installation
    dellemc/csi-vxflexos:v2.7.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.7.0
    dellemc/csi-vxflexos:v2.8.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.8.0
    dellemc/csi-vxflexos:v2.9.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.9.0
-   dellemc/sdc:3.6 -> localregistry:5000/csi-operator/sdc:3.6
    dellemc/sdc:3.6.0.6 -> localregistry:5000/csi-operator/sdc:3.6.0.6
```
Comment (gallacher) [inline content/docs/csidriver/installation/offline/_index.md:228]: What about the removal of the other dellemc/sdc:* values?

File: content/docs/csidriver/_index.md [hunk 4]
```diff
@@ -58,13 +58,9 @@ The CSI Drivers by Dell implement an interface between [CSI](https://kubernetes-
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerMax                                                | PowerFlex        | Unity XT                   | PowerScale                         |    PowerStore    |
 |---------------|:-------------------------------------------------------:|:----------------:|:--------------------------:|:----------------------------------:|:----------------:|
-| Storage Array | PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx <br>Unisphere 10.0,10.0.1 |    3.5.x, 3.6.x, 4.0.x, 4.5  | 5.1.x, 5.2.x, 5.3.0 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5.0.5, 9.5.0.6 | 2.0.x, 2.1.x, 3.0, 3.2, 3.5     |
+| Storage Array | PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx <br>Unisphere 10.0,10.0.1 |    3.6.x, 4.0.x, 4.5  | 5.1.x, 5.2.x, 5.3.0 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5.0.5, 9.5.0.6 | 2.0.x, 2.1.x, 3.0, 3.2, 3.5     |
```
Comment (gallacher) [inline content/docs/csidriver/_index.md:61]: This is being address as part of: https://github.com/dell/csm-docs/pull/885.  So you can revert this change.

**Review by AkshaySainiDell — APPROVED** [review]
LGTM

File: content/docs/csidriver/installation/offline/_index.md (hunk 3 above)
Comment (adarsh-dell) [reply content/docs/csidriver/installation/offline/_index.md]: Done.

File: content/docs/csidriver/_index.md (hunk 4 above)
Comment (adarsh-dell) [reply content/docs/csidriver/_index.md]: Done.

**Review by suryagupta4 — APPROVED** [review]

**Review by mareksuski-dell — APPROVED** [review]
LGTM
